src/main/java/com/redhat/patriot/network_simulator/example/files/FileUtils.java

```diff
@@ -0,0 +1,32 @@
+package com.redhat.patriot.network_simulator.example.files;
+
+import java.io.File;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.StandardCopyOption;
+
+public class FileUtils {
+    public String convertToFile(InputStream inputStream, String name) {
+        File targetFile = new File(name);
```

Comment:
And do you close this file? Wouldn't it be better to declare as resource in try block?

Reply:
I think file java is only representing path, name,... File is not closable. I can add InputStream to resources, but I don't think it's necessary.

Reply:
Agreed.